meson_options.txt

```diff
@@ -191,98 +131,171 @@ option(
     description: 'Use OS specific init config',
 )
 option(
-    'disable-init-hooks',
+    'with-init-hooks',
     type: 'boolean',
-    value: false,
-    description: 'Disable install hooks for OS specific init system',
+    value: true,
+    description: 'Enable install hooks for OS specific init system',
 )
 option(
     'with-kerberos',
-    type: 'feature',
-    value: 'auto',
-    description: 'Kerberos V support',
+    type: 'boolean',
+    value: true,
+    description: 'Enable Kerberos V API support',
+)
+option(
+    'with-krbV-uam',
+    type: 'boolean',
+    value: true,
+    description: 'Enable build of Kerberos V UAM module',
+)
+option(
+    'with-largefile',
+    type: 'boolean',
+    value: true,
+    description: 'Enable support for large files',
 )
 option(
     'with-ldap',
+    type: 'boolean',
+    value: true,
+    description: 'Enable LDAP support'
+)
+option(
+    'with-ldap-path',
     type: 'string',
     value: '',
-    description: 'LDAP support'
+    description: 'Specify path to LDAP library. Must contain lib and include dirs'
 )
 option(
-    'with-libgcrypt-dir',
+    'with-libgcrypt-path',
     type: 'string',
     value: '',
-    description: 'Path where LIBGCRYPT is installed (optional). Must contain lib and include dirs',
+    description: 'Specify path to libgcrypt installation. Must contain lib and include dirs',
 )
 option(
     'with-libiconv',
+    type: 'boolean',
+    value: true,
+    description: 'Enable iconv support',
+)
+option(
+    'with-libiconv-path',
     type: 'string',
     value: '',
-    description: 'Path to libiconv installation. Must contain lib and include dirs',
+    description: 'Specify path to libiconv installation. Must contain lib and include dirs',
 )
 option(
-    'with-lockfile',
+    'with-lockfile-path',
     type: 'string',
     value: '',
-    description: 'Path to Netatalk lockfile',
+    description: 'Specify path to Netatalk lockfile',
+)
+option(
+    'with-manual',
+    type: 'boolean',
+    value: false,
+    description: 'Specify path to Docbook XSL directory',
 )
 option(
     'with-mysql-config',
     type: 'string',
     value: 'mysql_config',
-    description: 'Path to mysql-config binary',
+    description: 'Specify name of mysql-config binary',
+)
+option(
+    'with-overwrite',
```

Comment:
Minor thing, but while adjusting option names, why not be more specific with this one and call it something like "with-overwrite-conf"? (As opposed to binaries or libraries etc.)